README.md

```diff
@@ -1 +1,130 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio Website <a name="about-project"></a>
+
+**Portfolio** is a website that i will use to showcase my projects at microverse and let other developers to know about myself.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML5</a></li>
+    <li><a href="#">CSS3</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **About Me**
+- **Recent work and Projects**
+- **Skills**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@jeanbuhendwa](https://github.com/jeanbuhendwa)
+- Twitter: [@jeanbuhendwa](https://twitter.com/jeanjacqueI)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/johnbuhendwa/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Desktop and Tablet Design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project don't hesitate to leave a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse of the good resources they provided in this program
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **What's the difference between Git and GitHub**
+
+  - Git is a version control system that tracks changes in project files while GitHub is hosting repository
+
+- **What's Gitflow**
+
+  - Gitflow is an alternative Git branching model that involves the use of feature branches and multiple primary branches.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
Please include the MIT licence in this project. Adding the licence to the project tells users what they can and cannot do with your project.

Reply:
- [ Optional ] I must commend that you did awesomely In giving a review with a friendly tune and suggesting a "why" on changes needed 👏 . As an improvement, please consider using friendly emojis as they also help in giving a warm feel to the review. 👏

- [Optional] Also, I strongly suggest you add a link to references. for example instead of just stating "MIT license" you can  add a hyperlink [MITLicense](https://www.MIT.com)